native_client/ctcdecode/swigwrapper.i

```diff
@@ -4,6 +4,7 @@
 #include "ctc_beam_search_decoder.h"
 #define SWIG_FILE_WITH_INIT
 #define SWIG_PYTHON_STRICT_BYTE_CHAR
+#include "workspace_status.cc"
```

Comment:
Why are you including the generated file here rather than the header?

Reply:
This way we don't need extra CC pass on the `workspace_status.cc` to put it

Reply:
Doh, I should not have wrote that code at that time. I get your point.